python/python/tests/test_async_client.py

```diff
@@ -7261,6 +7261,20 @@ async def test_copy_database(self, redis_client: GlideClient):
         finally:
             assert await redis_client.select(0) == OK
 
+    @pytest.mark.parametrize("cluster_mode", [True, False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_wait(self, redis_client: TGlideClient):
+        key = f"{{key}}-1{get_random_string(5)}"
+        value = get_random_string(5)
+        value2 = get_random_string(5)
+
+        assert await redis_client.set(key, value) == OK
+        assert await redis_client.wait(1, 1000) >= 0
+
+        # ensure that command doesn't time out even if timeout > request timeout (250ms by default)
+        assert await redis_client.set(key, value2) == OK
+        assert await redis_client.wait(100, 500) >= 0
+
```

Comment:
Should we add the following tests that also exist on the Java side?
- wrap a WAIT call with numreplicas=100 and timeout=0 in a task, ensure it raises a TimeoutError
- pass negative timeout value, ensure an error occurs

Reply:
added bad input test, the other test mainly checks that the command wouldnt time itself out even if the timeout is above the client request timeout, which getting a result from that line should suffice.

Reply:
Yeah I guess it also tests the no-timeout functionality but maybe that's not necessary really